apiserver/apiserver.go

```diff
@@ -712,6 +718,15 @@ func (srv *Server) endpoints() ([]apihttp.Endpoint, error) {
 		GetHandler:  modelCharmsHandler.ServeGet,
 	}
 	modelCharmsUploadAuthorizer := tagKindAuthorizer{names.UserTagKind}
+
+	modelS3CharmsHandler := &objectsCharmHandler{
+		ctxt: httpCtxt,
+	}
+	modelS3CharmsHTTPHandler := &objectsCharmHTTPHandler{
```

Comment:
I think we should standardize on the naming here S3 vs object (I vote the latter).

Reply:
Yeah, definitely agree! I had forgot to change some of the names.. should be OK now